src/main/java/minicraft/core/Renderer.java

```diff
@@ -281,7 +281,7 @@ private static void renderGui() {
 		if (player.activeItem instanceof ToolItem) {
 			// Draws the text
 			ToolItem tool = (ToolItem) player.activeItem;
-			int dura = tool.dur * 100 / (tool.type.durability * (tool.level+1));
```

Comment:
See, I find things like these problematic. Since now you just removed the static variable in the class and replaced it with a json variable.

Reply:
Hmm it is because it doesn't show error since it is dynamic. There won't be compiling error, runtime error instead, when there is error.

Reply:
That is not a good thing. It is better to get compiler errors than runtime errors.

Reply:
So...for item, dur and book shouldn't be dynamic.

Reply:
None of them needs to be dynamic. Just the same features could be added by making it a part of the class, in addition to more stability.

Reply:
Recovered the changes on this.